### PR TITLE
Create review cards, add new add product screen, update backend route to search by review text, add search functionality

### PR DESCRIPTION
**Pull request by mohaimenhasan**
**Check deployment of the code here - **

[Added the following:](https://github.com/mohaimenhasan/Rekko/commit/be07a681ccf0b8bd0768c8948e4b679a7d45114c) 

Search route by review
screens to show reviews
add a new review once the user is an existing user
fixed css for the card
search is working need to rethink the use of global states and render the reviews
review cards

Fix:
- Fixes issues with the URL
- Fixes heroku issues 
- Fixes some of the custom heroku issues

### CODE REVIEW

File: client/src/assets/css/home.css [hunk 1]
```diff
@@ -15,4 +15,16 @@
 .searchbar-container {
     margin-top: 31px;
     margin-bottom: 27px;
+}
+
+@media only screen and (max-width: 540px) {
```
Comment (arnav2) [inline client/src/assets/css/home.css:20]: just wondering could you provide a bit more information on why only these two width are required ?

File: client/src/components/ReviewCardContainer.js [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
+    const [state, setState] = useGlobalState({reviewCards: []});
+    
     useEffect(async () => {
         await axios.get(`${window.location.origin.toString()}/review/getAllReviews`, {})
         .then(res => {
             console.log(res.data.data);
             let allReviews = [];
-            res.data.data.forEach(x => allReviews.push(<ReviewCard heading={x['User']['UserLogin']} brandName={x['Product']['ProductBrand']} productName={x['Product']['ProductName']} review={x['ReviewText']}/>));
-            console.log(allReviews);
+            res.data.data.forEach(x => allReviews.push(<ReviewCard heading={`${x['User']['UserLogin']}, ${(new Date(x['ReviwedAt'])).toLocaleString('default', { month: 'short', day: 'numeric', year: 'numeric'})} at ${(new Date(x['ReviwedAt'])).toLocaleString('default', { timeStyle: 'long'})}`} brandName={x['Product']['ProductBrand']} productName={x['Product']['ProductName']} review={x['ReviewText']}/>));
```
Comment (arnav2) [inline client/src/components/ReviewCardContainer.js:14]: Can we make a separate function for the creating the Data. Something such as createDate function ?

**Review by arnav2 — COMMENTED** [review]
Just some minor questions and suggestions. Change looks good otherwise

File: client/src/assets/css/home.css (hunk 1 above)
Comment (mohaimenhasan) [reply client/src/assets/css/home.css:20]: On this is from doing browser testing. I can do a quick demo of why I chose these numbers

**Pull request merged**